compose/compose-file.md

```diff
@@ -415,6 +532,38 @@ Running `id` inside the created container will show that the user belongs to
 the `mail` group, which would not have been the case if `group_add` were not
 used.
 
+### healthcheck
+
+> [Version 3 file format](compose-file.md#version-3) and up.
+
+Configure a check that's run to determine whether or not containers for this
+service are "healthy". See the docs for the
+[HEALTHCHECK Dockerfile instruction](/engine/reference/builder.md#healthcheck)
+for details on how healthchecks work.
+
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost"]
+      interval: 1m30s
+      timeout: 10s
+      retries: 3
```

Comment:
Good catch, I've added a note at the bottom and removed the `test: ["NONE"]` example.